.github/workflows/cflite_pr.yml

```diff
@@ -0,0 +1,42 @@
+name: ClusterFuzzLite PR fuzzing
+on:
+  pull_request:
+    paths:
+      - '**'
+permissions: read-all
+jobs:
+  PR:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ matrix.sanitizer }}-${{ github.ref }}
+      cancel-in-progress: true
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer: [address, undefined, memory] # Override this with the sanitizers you want.
+    steps:
+    - name: Build Fuzzers (${{ matrix.sanitizer }})
+      id: build
+      uses: google/clusterfuzzlite/actions/build_fuzzers@v1
+      with:
+        language: c # Change this to the language you are fuzzing.
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The token is not declared in the other workflow (in the same `build` step). Is it an error, or the token is only useful for `storage-repo` option, and thus could be removed as long as storage feature is not used?

Reply:
The `github-token` is needed in the build step, since it does some github API requests, I've updated to include in both workflows